config/crd/bases/infrastructure.cluster.x-k8s.io_metal3datatemplates.yaml

```diff
@@ -56,6 +56,23 @@ spec:
                 description: MetaData contains the information needed to generate
                   the metadata secret
                 properties:
+                  dnsServersFromIPPool:
+                    description: DNSServersFromPool is the list of metadata items
+                      to be rendered as dns servers.
+                    items:
+                      properties:
+                        key:
+                          description: Key is the metadata key when redendering this
```

Comment:
"Key is the metadata key"
What is this key used for ? to retrieve the object ?
Mention the purpose.